tests/app/data/test_schema_id_regex.py

```diff
@@ -32,7 +31,7 @@ def validate_json_against_schema(json_to_validate, schema):
 class TestSchemaIdRegEx(unittest.TestCase):
 
     def setUp(self):
-        schema_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY, "schema/schema_v1.json"), encoding="utf8")
+        schema_file = open(_get_schema_file_path("schema/schema_v1.json", language_code='en'), encoding="utf8")
```

Comment:
I'd go back to using the original `os.path.join` as this method won't work for `schema_v1.json` and it doesn't seem right to add a method to our code for loading it when it's only used in the tests.